components/collective-navbar/index.js

```diff
@@ -586,6 +587,7 @@ const CollectiveNavbar = ({
                 collective={collective}
                 callsToAction={callsToAction}
                 hiddenActionForNonMobile={mainAction?.type}
+                mainAction={Object.values(pick(mainAction, 'type'))}
```

Comment:
I don't understand why we pass an array here. From the variable name I was expecting `mainAction` to be something like `SUBMIT_EXPENSE` or `CONTACT`

Reply:
It was so I could use the `difference` array method with `enabledCTAs` in the `ActionsMenu`

Reply:
`difference(enabledCTAs, [mainAction])` could be simpler in that case, though I would recommend `without` for that case:

```es6
without(enabledCTAs, mainAction)
```

https://lodash.com/docs/4.17.15#without